resources/newssite/news-nuxt/app.vue

```diff
@@ -16,51 +16,10 @@ if (process.client) {
 }
 </script>
 
-<script setup>
-import { nextTick, watch } from "#imports";
```

Comment:
What's up with these removals?

Reply:
I am using a layout file again - it was just easier to make the data available where I need it.